packages/fd/manifest.json

```diff
@@ -0,0 +1,6 @@
+{
+    "name": "Fast Downward",
+    "shortname": "fd",
```

Comment:
Also, could you rename `fd` to `fast-downward` or `downward`? "FD" is not an "official abbreviation".

Reply:
Hehe, ok. I like `fd`, by I can just alias my own system...